src/leiningen/new/re_frame/project.clj

```diff
@@ -17,14 +19,19 @@
                                     "test/js"{{/test?}} {{#garden?}}
                                     "resources/public/css/compiled"{{/garden?}}]
 
```

Comment:
Ok I did that, also added a base CSS file if garden is not used. How does that look?